### PR TITLE
Add note about changing the default value of  properties.message.time…

### LINKED ISSUE #2533: Document: feat(sink): add es retry_on_conflict and max_task_num
This issue tracks the documentation update needed for the merged PR #17867.

Source PR URL: https://github.com/risingwavelabs/risingwave/pull/17867
Source PR Merged At: 2024-08-29T03:54:40Z

If it is a major improvement that deserves a new page or a new section in the documentation, please check if we should label it as an experiment feature.

### LINKED ISSUE #2534: Document: feat(sink): use official default message_timeout_ms
This issue tracks the documentation update needed for the merged PR #18304.

Source PR URL: https://github.com/risingwavelabs/risingwave/pull/18304
Source PR Merged At: 2024-08-29T05:26:42Z

If it is a major improvement that deserves a new page or a new section in the documentation, please check if we should label it as an experiment feature.

### PR DESCRIPTION
**Pull request by ShanlanLi**
…out.ms

## Related code PR
https://github.com/risingwavelabs/risingwave/pull/18304

## Related doc issue

Resolve https://github.com/risingwavelabs/risingwave-docs/issues/2533

## Checklist

- [ ] I have checked the doc site preview, and the updated parts look good.
- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`emile-00`, `hengm3467`, & `WanYixian`).

### CODE REVIEW

File: docs/guides/create-sink-kafka.md [hunk 1]
```diff
@@ -73,7 +73,9 @@ When creating a Kafka sink in RisingWave, you can specify the following Kafka-sp
 |ssl.endpoint.identification.algorithm | properties.ssl.endpoint.identification.algorithm | str |
 
 :::note
-Set `properties.ssl.endpoint.identification.algorithm` to `none` to bypass the verification of CA certificates and resolve SSL handshake failure. This parameter can be set to either `https` or `none`. By default, it is `https`.
++ Set `properties.ssl.endpoint.identification.algorithm` to `none` to bypass the verification of CA certificates and resolve SSL handshake failure. This parameter can be set to either `https` or `none`. By default, it is `https`.
+
++ Starting with version 2.0, the default value for `properties.message.timeout.ms` has changed. It's now **5 minutes** instead of 5 seconds, matching the default setting in the official Kafka library.
```
Comment (ShanlanLi) [inline docs/guides/create-sink-kafka.md]: @wenym1 The detailed description for this parameter is in [the repo file](https://github.com/confluentinc/librdkafka/blob/master/CONFIGURATION.md). Not sure if it should be changed too.
image

Comment (wenym1) [reply docs/guides/create-sink-kafka.md]: Yes, please link `official Kafka library` to the repo file.

**Review by wenym1 — APPROVED** [review]

File: docs/guides/create-sink-kafka.md (hunk 1 above)
Comment (WanYixian) [inline docs/guides/create-sink-kafka.md]:
```suggestion
+ Starting with version 2.0, the default value for `properties.message.timeout.ms` has changed from 5 seconds to **5 minutes**, aligning with the default setting in the official Kafka library.
```

**Review by WanYixian — APPROVED** [review]
LGTM

File: docs/guides/create-sink-kafka.md [hunk 2]
```diff
@@ -73,7 +73,9 @@ When creating a Kafka sink in RisingWave, you can specify the following Kafka-sp
 |ssl.endpoint.identification.algorithm | properties.ssl.endpoint.identification.algorithm | str |
 
 :::note
-Set `properties.ssl.endpoint.identification.algorithm` to `none` to bypass the verification of CA certificates and resolve SSL handshake failure. This parameter can be set to either `https` or `none`. By default, it is `https`.
++ Set `properties.ssl.endpoint.identification.algorithm` to `none` to bypass the verification of CA certificates and resolve SSL handshake failure. This parameter can be set to either `https` or `none`. By default, it is `https`.
+
++ Starting with version 2.0, the default value for `properties.message.timeout.ms` has changed from 5 seconds to **5 minutes**, aligning with the default setting in the official Kafka library.
```
Comment (ShanlanLi) [inline docs/guides/create-sink-kafka.md]:
```suggestion
+ Starting with version 2.0, the default value for `properties.message.timeout.ms` has changed from 5 seconds to **5 minutes**, aligning with the default setting in the [official Kafka library](https://github.com/confluentinc/librdkafka/blob/master/CONFIGURATION.md).
```

**Pull request merged**